### PR TITLE
xkbcomp: add xkbcomp package

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/xorg-app/xkbcomp_debian.bb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# depends on virtual/libx11
+REQUIRED_DISTRO_FEATURES = "x11"
+
+inherit debian-package
+require recipes-debian/sources/x11-xkb-utils.inc
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/x11-xkb-utils.git"
+S = "${DEBIAN_UNPACK_DIR}/xkbcomp"
+PV = "1.4.0"
```
Comment (masami256) [inline recipes-debian/xorg-app/xkbcomp_debian.bb]: @yuwata  Do we need change PV from 7.7+4(is defined in recipes-debian/sources/x11-xkb-utils.inc) to 1.4.0?

Comment (yuwata) [reply recipes-debian/xorg-app/xkbcomp_debian.bb]: Yes. I will add a comment about that. Thanks.

Comment (yuwata) [reply recipes-debian/xorg-app/xkbcomp_debian.bb]: A comment is added. PTAL.

Comment (masami256) [reply recipes-debian/xorg-app/xkbcomp_debian.bb]: thanks!

**Pull request merged**